app/assets/stylesheets/components/_action.scss

```diff
@@ -7,7 +7,7 @@
   padding-top: 0.9375em;
   padding-bottom: 0.625em;
 
-  a:first-child {
+  a.eui-btn--blue:first-child {
```

Comment:
I'd rather be less specific here and make sure we catch all buttons. `a[class^="eui-btn"]:first-child`

Reply:
+1 to this. If you're wondering what the heck that selector does... check this out number 13 at this link: https://code.tutsplus.com/tutorials/the-30-css-selectors-you-must-memorize--net-16048

I don't think you need to memorize all 30 of those selectors, but I reference this list all the time. It's gold.